client/settings/fraud-protection/advanced-settings/index.tsx

```diff
@@ -237,9 +245,51 @@ const FraudProtectionAdvancedSettingsPage: React.FC = () => {
 		};
 	}, [ isLoading ] );
 
+	const { isFRTReviewFeatureActive } = wcpaySettings;
+
+	const confirmLeaveCallback = useConfirmNavigation( () => {
+		const settingsChanged =
+			! isLoading &&
+			! isMatchWith(
+				readRuleset( advancedFraudProtectionSettings ),
+				protectionSettingsUI,
+				( source, target ) => {
+					for ( const rule in source ) {
+						// We need to skip checking the "block" property, as they are not the same with defaults.
+						if ( ! isFRTReviewFeatureActive && rule === 'block' ) {
```

Comment:
When the FRT review feature is disabled, we don't care about what the block type is, so skip checking those. The saved data on the client and the UI showed differences and couldn't be changed because FRT review feature was disabled, so I needed to skip testing the object property for dirty checks.